github-profiles/AlexMGonzalez-6883/README.md

```diff
@@ -0,0 +1,66 @@
+https://github.com/alexgonza278
```

Comment:
Congratulations on doing a great job! .

Perhaps you could add this link in a better way and it could be considered for placement at the end of this file.
Here's an example for your reference, I hope you find it helpful:
[![Github](https://img.shields.io/badge/Github-%2324292e.svg?logo=github&logoColor=white)](github-profiles/AlexMGonzalez-6883/README.md)